wiki/Medals/en.md

```diff
@@ -1,59 +1,295 @@
+| Chosen | Full combo any Qualified, Ranked, or Approved beatmap that lies in the 9.00★–9.99★ star rating range (after all the [difficulty increase mods](/wiki/Game_modifier#difficulty-increase), if any, have been factored in) in the osu! game mode. Any usage of [difficulty reduction mods](/wiki/Game_modifier#difficulty-reduction) is not permitted. |
+| Unfathomable | Full combo any Qualified, Ranked, or Approved beatmap that lies in the 10.00★–10.99★ star rating range (after all the [difficulty increase mods](/wiki/Game_modifier#difficulty-increase), if any, have been factored in) in the osu! game mode. Any usage of [difficulty reduction mods](/wiki/Game_modifier#difficulty-reduction) is not permitted. |
 
----
+## Dedication
 
-The High Ranker medals can be obtained by reaching a certain global ranking position in **any** game mode.
+The **Dedication medals** are medals that are being awarded upon reaching certain playcount milestones in a single game mode.
 
----
+### List of obtainable Dedication medals
 
-The star-based medals are split into two groups, *Pass* and *S*. The *Pass* medals can be obtained by passing a beatmap of a certain star difficulty (excluding EZ/NF/HT). The *S* medals will be granted if you meet the conditions of the *Pass* medal with a *S* ranking whilst also having an [FC](/wiki/FC). The difficulty of the beatmap will be rounded to the nearest star difficulty and will grant the medal for what it is rounded to. For example, if you have not earned the 6/7 star medals but cleared 7.8 stars difficulty, you will only receive a 7 star medal. You can also use difficulty increasing mods, such as HR/DT (based on increased stars). The beatmap must also be in the Ranked state to grant a medal.
+| Medal name | Requirement(s) |
+| :-- | :-- |
+| 5,000 Plays | Achieve a total of 5,000 playcounts in any game mode. |
+| 15,000 Plays | Achieve a total of 15,000 playcounts in any game mode. |
+| 25,000 Plays | Achieve a total of 25,000 playcounts in any game mode. |
+| 50,000 Plays | Achieve a total of 50,000 playcounts in any game mode. |
 
-## Dedication
+## Mod Introduction
 
-The **Dedication** medals can be obtained by playing a lot of osu!standard, collecting many fruits in osu!catch, hitting getting a lot of drum hits in osu!taiko, by pressing a lot of keys in osu!mania.
+*See also: [Game modifier](/wiki/Game_modifier)*
 
-## Mod Introduction
+The **Mod Introduction medals** are medals that are being awarded by clearing any [Qualified](/wiki/Beatmap/Category#qualified), [Ranked](/wiki/Beatmap/Category#ranked), or [Approved](/wiki/Beatmap/Category#approved) beatmap with the corresponding game modifier. These medals are meant to encourage newer players to try out each mod and explore what the game has to offer. Do note that however if the player plays with multiple game modifiers at a time, the player is **not** eligible to earn *any* of the game modifier medals for that play.
```

Comment:
```suggestion
*Mod Introduction medals* are medals that are awarded upon clearing any [Qualified](/wiki/Beatmap/Category#qualified), [Ranked](/wiki/Beatmap/Category#ranked), or [Approved](/wiki/Beatmap/Category#approved) beatmap with their corresponding game modifier. These medals are meant to encourage newer players to try out each mod and explore what the game has to offer. Do note that, however, that if players play with multiple game modifiers at a time, the player is **not** eligible to earn *any* of the game modifier medals for that play.
```
probably shouldn't flip flop between "game modifier" and "mod" here. consider this:
```suggestion
*Mod Introduction medals* are medals that are awarded upon clearing any [Qualified](/wiki/Beatmap/Category#qualified), [Ranked](/wiki/Beatmap/Category#ranked), or [Approved](/wiki/Beatmap/Category#approved) beatmap with their corresponding game modifier (mod). These medals are meant to encourage newer players to try out each mod and explore what the game has to offer. Do note that, however, that if players play with multiple mods at a time, the player is **not** eligible to earn *any* of the game modifier medals for that play.
```

Reply:
did a mix of both